vendor/assets/javascripts/jquery.minicolors.simple_form.js

```diff
@@ -1,6 +1,6 @@
-$(function() {
-  $('[data-minicolors]').each(function(i, elem) {
-    var input = $(this);
-    input.minicolors(input.data('minicolors'));
-  });
+$(document).on('turbolinks:load', function(e) {
```

Comment:
This will indeed listen to the event `turbolinks:load`.
But in case there are no Turbolinks (Turbolinks disabled), this won't do anything and will break every application using jQuery without Turbolinks.